src/Block/Renderer/DocumentRenderer.php

```diff
@@ -21,9 +21,9 @@
 class DocumentRenderer implements BlockRendererInterface
 {
     /**
-     * @param Document $block
+     * @param AbstractBlock|Document $block
+     * @param HtmlRendererInterface $htmlRenderer
      * @param bool $inTightList
-     *
```

Comment:
Could we keep this empty line between the `param` and `return`?  I like using the PEAR standard of having exactly one empty line in between.